### ISSUE TITLE
Graphics Device initialization failed for :  es2, sw Error initializing QuantumRenderer: no suitable pipeline found

### ISSUE THREAD

**Issue by atultiwariatul** [openjfx/openjfx-docs#243]
I am trying to Run JavaFX project with below Configuration 
1. IntelliJ IDEA 2023.1
2. Open JDK 21
3. javafx-sdk-21.0.1 for MAC AMD64 Bit

VM Options are set with JavaFX SDK's lib folder with --add-module but while Running the APP I am getting 

/Library/Java/JavaVirtualMachines/jdk-21.0.1.jdk/Contents/Home/bin/java --module-path /javafx-sdk-21.0.1/lib/ --add-modules=javafx.fxml,javafx.controls,javafx.graphics -Djava.library.path=/javafx-sdk-21.0.1/lib -javaagent:/Applications/IntelliJ IDEA.app/Contents/lib/idea_rt.jar=52740:/Applications/IntelliJ IDEA.app/Contents/bin -Dfile.encoding=UTF-8 -Dsun.stdout.encoding=UTF-8 -Dsun.stderr.encoding=UTF-8 -classpath /Users/atultiwari/Projects/JavaFX/LSQLApp/out/production/LSQLApp:/javafx-sdk-21.0.1/lib/javafx-swt.jar:/javafx-sdk-21.0.1/lib/javafx.web.jar:/javafx-sdk-21.0.1/lib/javafx.base.jar:/javafx-sdk-21.0.1/lib/javafx.fxml.jar:/javafx-sdk-21.0.1/lib/javafx.media.jar:/javafx-sdk-21.0.1/lib/javafx.swing.jar:/javafx-sdk-21.0.1/lib/javafx.controls.jar:/javafx-sdk-21.0.1/lib/javafx.graphics.jar:/javafx-sdk-21.0.1/controlsfx-11.1.2.jar:/javafx-sdk-21.0.1/fontawesomefx-8.2.jar:/javafx-sdk-21.0.1/fontawesomefx-8.9.jar:/javafx-sdk-21.0.1/fontawesomefx-commons-9.1.2.jar:/javafx-sdk-21.0.1/fontawesomefx-fontawesome-4.7.0-9.1.2.jar:/javafx-sdk-21.0.1/fontawesomefx-icons525-4.2.0-9.1.2.jar:/javafx-sdk-21.0.1/fontawesomefx-materialdesignfont-2.0.26-9.1.2.jar:/javafx-sdk-21.0.1/fontawesomefx-materialicons-2.2.0-9.1.2.jar:/javafx-sdk-21.0.1/fontawesomefx-octicons-4.3.0-9.1.2.jar:/javafx-sdk-21.0.1/jfoenix-9.0.1.jar:/javafx-sdk-21.0.1/webcam-capture-0.3.12.jar com.houarizegai.learnsql.java.App
Graphics Device initialization failed for :  es2, sw
Error initializing QuantumRenderer: no suitable pipeline found
java.lang.RuntimeException: java.lang.RuntimeException: Error initializing QuantumRenderer: no suitable pipeline found
	at javafx.graphics@21.0.1/com.sun.javafx.tk.quantum.QuantumRenderer.getInstance(QuantumRenderer.java:283)
	at javafx.graphics@21.0.1/com.sun.javafx.tk.quantum.QuantumToolkit.init(QuantumToolkit.java:253)
	at javafx.graphics@21.0.1/com.sun.javafx.tk.Toolkit.getToolkit(Toolkit.java:263)
	at javafx.graphics@21.0.1/com.sun.javafx.application.PlatformImpl.startup(PlatformImpl.java:290)
	at javafx.graphics@21.0.1/com.sun.javafx.application.PlatformImpl.startup(PlatformImpl.java:162)
	at javafx.graphics@21.0.1/com.sun.javafx.application.LauncherImpl.startToolkit(LauncherImpl.java:651)
	at javafx.graphics@21.0.1/com.sun.javafx.application.LauncherImpl.launchApplicationWithArgs(LauncherImpl.java:409)
	at javafx.graphics@21.0.1/com.sun.javafx.application.LauncherImpl.launchApplication(LauncherImpl.java:364)
	at java.base/jdk.internal.reflect.DirectMethodHandleAccessor.invoke(DirectMethodHandleAccessor.java:103)
	at java.base/java.lang.reflect.Method.invoke(Method.java:580)
	at java.base/sun.launcher.LauncherHelper$FXHelper.main(LauncherHelper.java:1135)
Caused by: java.lang.RuntimeException: Error initializing QuantumRenderer: no suitable pipeline found
	at javafx.graphics@21.0.1/com.sun.javafx.tk.quantum.QuantumRenderer$PipelineRunnable.init(QuantumRenderer.java:95)
	at javafx.graphics@21.0.1/com.sun.javafx.tk.quantum.QuantumRenderer$PipelineRunnable.run(QuantumRenderer.java:125)
	at java.base/java.lang.Thread.run(Thread.java:1583)
Exception in thread "main" java.lang.reflect.InvocationTargetException
	at java.base/jdk.internal.reflect.DirectMethodHandleAccessor.invoke(DirectMethodHandleAccessor.java:118)
	at java.base/java.lang.reflect.Method.invoke(Method.java:580)
	at java.base/sun.launcher.LauncherHelper$FXHelper.main(LauncherHelper.java:1135)
Caused by: java.lang.RuntimeException: No toolkit found
	at javafx.graphics@21.0.1/com.sun.javafx.tk.Toolkit.getToolkit(Toolkit.java:275)
	at javafx.graphics@21.0.1/com.sun.javafx.application.PlatformImpl.startup(PlatformImpl.java:290)
	at javafx.graphics@21.0.1/com.sun.javafx.application.PlatformImpl.startup(PlatformImpl.java:162)
	at javafx.graphics@21.0.1/com.sun.javafx.application.LauncherImpl.startToolkit(LauncherImpl.java:651)
	at javafx.graphics@21.0.1/com.sun.javafx.application.LauncherImpl.launchApplicationWithArgs(LauncherImpl.java:409)
	at javafx.graphics@21.0.1/com.sun.javafx.application.LauncherImpl.launchApplication(LauncherImpl.java:364)
	at java.base/jdk.internal.reflect.DirectMethodHandleAccessor.invoke(DirectMethodHandleAccessor.java:103)
	... 2 more

**Comment by atultiwariatul** [comment]
I am not sure if this solves the issue but what I did is;
1. Made Sure my processor is Intel and hence it's using X64 Architecture
2. My java_home was not set as I was directly using the JDK from IntellijIdea so I set JAVA_HOME ENV via .bash_profile
3. Downloaded again JavaFX (https://download2.gluonhq.com/openjfx/21.0.1/openjfx-21.0.1_osx-x64_bin-sdk.zip)
4. In Intellij Idea setup It's path via Project Settings-> Libraries and added path until <FX Directory>/lib
5. Restarted the System to make sure JAVA_HOME is properly propogated.

Ran the Program and It does solved my problem.
